.github/workflows/backend-tests.yml

```
@@ -37,11 +37,11 @@ jobs:
           go mod tidy -go=${{ env.GO_VERSION }}
           git diff --exit-code
 
+      - name: Install golangci-lint
+        run: go install github.com/golangci/golangci-lint/v2/cmd/golangci-lint@v2.13.1
+
       - name: Run golangci-lint
-        uses: golangci/golangci-lint-action@v9
-        with:
-          version: v2.13.1
-          args: --timeout=3m
+        run: $(go env GOPATH)/bin/golangci-lint run --timeout=3m
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:1: Quote this to prevent word splitting

(shellcheck)